### ISSUE TITLE
ManagePackageLicenses permission cannot be retrieved

### ISSUE THREAD

**Issue by jfaderanga** [forcedotcom/cli#2578]
### Summary
Retrieving permission set does not include `ManagePackageLicenses` permission. 

### Steps To Reproduce
- Create a permission set with license of Salesforce or Salesforce API Integration
- Assign the `Manage Package Licenses` Permission under System Permissions settings.
- Click save and retrieve via SF or SFDX CLI

### Expected result
- The permission set that was retrieved should have the `ManagePackageLicenses` permission in the XML

### Actual result
- The permission set that was retrieved does not have the `ManagePackageLicenses` permission in the XML

### System Information
MAC OS zsh
```
{
  "architecture": "darwin-arm64",
  "cliVersion": "@salesforce/cli/2.17.14",
  "nodeVersion": "node-v20.9.0",
  "osVersion": "Darwin 23.1.0",
  "rootPath": "/Users/jaysonf/.local/share/sf/client/2.17.14-31db5eb",
  "shell": "zsh",
  "pluginVersions": [
    "@oclif/plugin-autocomplete 3.0.1 (core)",
    "@oclif/plugin-commands 3.0.6 (core)",
    "@oclif/plugin-help 6.0.5 (core)",
    "@oclif/plugin-not-found 3.0.3 (core)",
    "@oclif/plugin-plugins 4.1.7 (core)",
    "@oclif/plugin-search 1.0.6 (core)",
    "@oclif/plugin-update 4.1.3 (core)",
    "@oclif/plugin-version 2.0.6 (core)",
    "@oclif/plugin-warn-if-update-available 3.0.2 (core)",
    "@oclif/plugin-which 3.0.8 (core)",
    "@salesforce/cli 2.17.14 (core)",
    "apex 2.3.20 (core)",
    "auth 2.8.26 (core)",
    "data 2.6.3 (core)",
    "deploy-retrieve 1.20.0 (core)",
    "info 3.0.1 (core)",
    "limits 3.0.1 (core)",
    "login 2.0.1 (core)",
    "marketplace 1.0.2 (core)",
    "org 2.11.8 (core)",
    "packaging 1.27.1 (user)",
    "schema 3.0.1 (core)",
    "settings 2.0.2 (core)",
    "sobject 0.2.15 (core)",
    "source 2.11.0 (core)",
    "telemetry 3.1.0 (core)",
    "templates 55.5.18 (core)",
    "trust 3.0.2 (core)",
    "user 2.3.42 (core)",
    "sfdx-git-delta 5.30.0 (user)",
    "sfpowerkit 6.1.0 (user)"
  ]
}
```
### Additional Information
- Same issue for sandbox and production environment.
- The xml file is deployable and updates the permission properly when you manually add the `ManagePackageLicenses` in the xml.
```
<userPermissions>
    <enabled>true</enabled>
    <name>ManagePackageLicenses</name>
</userPermissions>
```
- CLI Version: @salesforce/cli/2.6.9 darwin-arm64 node-v18.15.0

**Comment by mshanemc** [comment]
@jfaderanga I don't see that in the metdata API type for PermissionSet.  https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_permissionset.htm

What version of the metadata API supports ManagePackageLicenses?

**Comment by jfaderanga** [comment]
@mshanemc yeah I don't see it in the documentation as well. It works fine on API V59.0 if you manually added it.
```
<userPermissions>
    <enabled>true</enabled>
    <name>ManagePackageLicenses</name>
</userPermissions>
```

**Comment by mshanemc** [comment]
ok, if it deploys than we'll assume it's OK in the API and the docs are hopefully just behind.

Does the command show that it's retrieving v59 metadata (on a v59 connection?).  If not, that could be a setting in your `sfdx-project.json`.  

If so, can you you try to do this with some other mdapi tool (for example, workbench)?  I'd like to know if the API is not returning properly.  You could also try use the mdapi-flavored version of the command `--target-metadata-dir` and see if that brings it down.

I can tell you that the CLI isn't doing anything "inside" the xml contents on PermissionSet, so if the API is really omitting those, you'll have to open a case to get it fixed.

**Comment by jfaderanga** [comment]
yes, I tried multiple API version from 55.0 and it's the same output. I also tried the `--target-metadata-dir` on `sf project retrieve start` and same, it's not pulling it down..

**Comment by mshanemc** [comment]
Alrighty.  I'll turn this into a bug and assign it to the right team, but cases help with prioritization.

**Comment by mkhokhlov** [comment]
Any updates?
